### PR TITLE
doc: fix format 2 howto to suggest to declare a build export dependency on `message_runtime`

### PR DESCRIPTION
**Pull request by NikolausDemmel**
See https://github.com/ros/catkin/pull/790#issuecomment-202632124

### CODE REVIEW

File: doc/howto/format2/building_msgs.rst [hunk 1]
```diff
@@ -8,10 +8,11 @@ package.xml
 :::::::::::
 
 Your ``package.xml`` must declare a ``<build_depend>`` on
-``message_generation``, and a ``<exec_depend>`` on
-``message_runtime``::
+``message_generation``, and a ``<exec_depend>`` as well as
+``<build_export_depend>`` on ``message_runtime``::
```
Comment (dirk-thomas) [inline doc/howto/format2/building_msgs.rst]: Can you switch the order to be consistent with the following xml: `<build_export_depend> as well as <exec_depend>`

Comment (NikolausDemmel) [inline doc/howto/format2/building_msgs.rst]: Makes sense; fixed.

**Comment by dirk-thomas** [comment]
Thanks.

**Comment by dirk-thomas** [comment]
Cherry-picked to _kinetic-devel_ branch: 1b2e47643d782e147c17ecb8134a534a3f3630ef

**Comment by NikolausDemmel** [comment]
Thanks :+1:

**Pull request merged**